index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Microverse-Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <div class="con1">
+        <h1 class="logo-title">My Logo</h1>
+        <nav>
+          <img class="ham-img" src="/images/ham-menu.png" alt="Menu">
+        </nav>
+      </div>
+    </header>
+
+
+    <section class="Homepage">
+      <h2 class="hero-title">I’m Steve
+        Glad to see you!</h2>
```

Comment:
- [ ] Hi, the requirements state that this should be a `h1` tag. please implement this as a `h1` tag instead of `h2`.

something like this would work:
```suggestion
      <h1 class="hero-title">I’m Steve
        Glad to see you!</h1>
```